src/lib/config.js

```diff
@@ -1,14 +1,14 @@
-const fs = require('fs-extra');
-const findPrefix = require('find-npm-prefix');
-const os = require('os');
-const { prompt } = require('enquirer');
-const { table, getBorderCharacters } = require('table');
-const { step } = require('./helpers');
-const { green, red, bold, gray } = require('chalk');
-const Client = require('mina-signer');
-const { prompts } = require('./prompts');
-const { PrivateKey, PublicKey } = require('o1js');
-const HOME_DIR = os.homedir();
+import chalk from 'chalk';
+import enquirer from 'enquirer';
+import findPrefix from 'find-npm-prefix';
+import fs from 'fs-extra';
+import Client from 'mina-signer';
+import { PrivateKey, PublicKey } from 'o1js';
+import { getBorderCharacters, table } from 'table';
+import Constants from './constants.js';
```

Comment:
Nit Suggestion:  You could import the constants directly here and access them throughout the file instead of  `Constants.uiTypes`

Reply:
I was thinking that having an explicit object name here will improve readability, but I might be wrong so will apply requested changes.

Reply:
@ymekuria by the way, it won't work because `constants.js` module does not provide named export. Do you want me to change module and provide named exports instead?

Reply:
Or as an alternative we can leave object importing as is but extract required constant into separate variables at the top of the module. But it probably doesn't have much sense.

Reply:
That's fine to leave it as is.

Reply:
Let's leave it as is (no named exports).